json-formatter/CHANGELOG.md

```diff
@@ -13,6 +13,10 @@ This document is formatted according to the principles of [Keep A CHANGELOG](htt
 
 ### Changed
 
+* Handle the `JavaMethod` and `JavaStackTraceElement` fields in `SourceReference`:
+  - outputs `<className>.<methodName>` for `JavaMethod`
+  - outputs `<fileName>:<line?>` for `JavaStackTraceElement`
```

Comment:
Should be `<className>.<methodName>(<fileName>:<line>)`

Reply:
The `JavaStackTraceElement` message (your addition) doesn't have parameter information, so I can't add this at the moment.